src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+   */
+  label?: {
+    id?: string;
+    className?: string;
+    children?: React.ReactNode;
+  };
+  /**
+   * Provide hint to each checkbox item.
+   */
+  hint?: HintProps;
+  /**
+   * Divider text to separate radio items, for example the text "or".
+   */
+  divider?: string;
+  /**
+   * If true, radio will be checked.
+   */
+  checked?: boolean;
+  /**
+   * If true, content provided will be revealed when the item is checked.
+   */
+  conditional?: {
+    html: React.ReactNode;
+  };
+  /**
+   * If true, radio will be disabled.
+   */
+  disabled?: boolean;
+}
+
+export interface RadiosProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * Classes to add to the radio container.
+   */
+  className?: string;
+  /**
+   * Name attribute for each radio item.
+   */
+  name: string;
+  /**
+   * Name attribute for each radio item.
+   */
+  items: RadioButton[];
+  /**
+   * Options for the fieldset component (e.g. legend).
+   */
+  fieldset?: {
+    id?: string;
+    className?: string;
+    role?: string;
+    legend?: React.ReactNode;
+    describedBy?: string;
+  };
+  /**
+   * Options for the hint component (e.g. text).
+   */
+  hint?: HintProps;
+  /**
+   * Options for the errorMessage component (e.g. text).
+   */
+  errorMessage?: ErrorMessageProps;
+  /**
+   * Options for the form-group wrapper
+   */
+  formGroup?: {
+    id?: string;
+    className?: string;
+    error?: boolean;
+  };
+  /**
+   * String to prefix id for each checkbox item if no id is specified on each item.
+   * If `idPrefix` is not passed, fallback to using the name attribute instead.
+   */
+  idPrefix?: string;
+
+  onChange?(value: string): void;
+}
+
+export const Radios = (props: RadiosProps): JSX.Element => {
+  const {
+    className,
+    name,
+    items,
+    fieldset,
+    hint: radiosHint,
+    errorMessage,
+    onChange
+  } = props;
+  let formGroup = props.formGroup;
+  const idPrefix = props.idPrefix ? props.idPrefix : name;
+  let describedBy =
+    fieldset && fieldset.describedBy ? fieldset.describedBy : "";
+  let hintComponent: React.ReactNode, errorMessageComponent: React.ReactNode;
+  let isConditional = false;
+  for (const radio of items) {
+    if (radio.conditional && radio.conditional.html) {
+      isConditional = true;
+    }
+  }
+
+  if (radiosHint) {
+    const hintId = `${idPrefix}-hint`;
+    describedBy += ` ${hintId}`;
+    hintComponent = <Hint id={hintId} {...radiosHint} />;
+  }
+
+  if (errorMessage) {
+    const errorId = `${idPrefix}-error`;
+    describedBy += ` ${errorId}`;
+    errorMessageComponent = <ErrorMessage id={errorId} {...errorMessage} />;
+    if (formGroup) {
+      formGroup.error = true;
+    } else {
+      formGroup = { error: true };
+    }
+  }
+
+  const extraAttributes = Attributes.ariaAndData(props);
+  let classNames = "govuk-radios lbh-radios";
+  classNames += className ? ` ${className}` : "";
+  if (isConditional) {
+    classNames += " govuk-radios--conditional";
+  }
```

Comment:
There's a very useful library for this: `classnames`, which you can see examples of elsewhere. In this case, you could express this as:
```js
import classNames from "classnames";

const className = classNames("govuk-radios lbh-radios", className, { "govuk-radios--conditional": isConditional })
```